### PR TITLE
net-vpn/vtun: Port to EAPI 7

### PR DESCRIPTION
**Pull request by jsmolic**
Closes: https://bugs.gentoo.org/708372

### CODE REVIEW

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild [hunk 1]
```diff
@@ -14,32 +14,29 @@ SLOT="0"
 KEYWORDS="~alpha amd64 ppc ~sparc x86"
 IUSE="lzo socks5 ssl zlib"
 
-RDEPEND="ssl? ( dev-libs/openssl:0 )
+RDEPEND="
+	ssl? ( dev-libs/openssl:0 )
```
Comment (SoapGentoo) [inline net-vpn/vtun/vtun-3.0.3-r1.ebuild:18]: checked whether it links?

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild [hunk 2]
```diff
@@ -14,32 +14,29 @@ SLOT="0"
...
 	lzo? ( dev-libs/lzo:2 )
 	zlib? ( sys-libs/zlib )
 	socks5? ( net-proxy/dante )"
-DEPEND="${RDEPEND}
-	sys-devel/bison"
+DEPEND="${RDEPEND}"
+BDEPEND="sys-devel/bison"
 
 DOCS="ChangeLog Credits FAQ README README.Setup README.Shaper TODO"
```
Comment (SoapGentoo) [inline net-vpn/vtun/vtun-3.0.3-r1.ebuild:25]: turn into array

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild [hunk 3]
```diff
@@ -56,4 +53,5 @@ src_install() {
 	newinitd "${FILESDIR}"/vtun.rc vtun
 	insinto etc
```
Comment (SoapGentoo) [inline net-vpn/vtun/vtun-3.0.3-r1.ebuild:54]: `insinto /etc` is the style we prefer in general

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild [hunk 4]
```diff
@@ -14,32 +14,29 @@ SLOT="0"
...
+	"${FILESDIR}"/${PN}-libssl-1.1.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+)
+
 src_prepare() {
-	sed -i Makefile.in \
-		-e '/^LDFLAGS/s|=|+=|g' \
-		|| die "sed Makefile"
-	eapply "${FILESDIR}"/${P}-includes.patch
-	# remove unneeded checking for /etc/vtund.conf
```
Comment (SoapGentoo) [inline net-vpn/vtun/vtun-3.0.3-r1.ebuild:33]: put these comments in the `PATCHES` array

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild (hunk 1 above)
Comment (jsmolic) [reply net-vpn/vtun/vtun-3.0.3-r1.ebuild]: Yes

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild (hunk 3 above)
Comment (jsmolic) [reply net-vpn/vtun/vtun-3.0.3-r1.ebuild]: Sure, done

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild (hunk 4 above)
Comment (jsmolic) [reply net-vpn/vtun/vtun-3.0.3-r1.ebuild:33]: Done

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild [hunk 5]
```diff
@@ -14,32 +14,32 @@ SLOT="0"
 KEYWORDS="~alpha amd64 ppc ~sparc x86"
```
Comment (SoapGentoo) [inline net-vpn/vtun/vtun-3.0.3-r1.ebuild:14]: revbump, too invasive

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: net-vpn/vtun/vtun-3.0.3-r1.ebuild (hunk 5 above)
Comment (jsmolic) [reply net-vpn/vtun/vtun-3.0.3-r1.ebuild]: done

**Pull request closed without merging**